examples/vector_tile_3d_mesh.html

```diff
@@ -0,0 +1,104 @@
+        <script src="../dist/itowns.js"></script>
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script>
+            // # Simple Globe viewer + a vector tile layer
+
+            // Define initial camera position
+            var placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.351323, 48.856712),
+                range: 5000,
+                tilt: 45,
+            }
+            var promises = [];
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView*
+            var view = new itowns.GlobeView(viewerDiv, placement);
+
+            // define pole texture
+            view.tileLayer.noTextureColor = new itowns.THREE.Color(0x95c1e1);
+
+            view.getLayerById('atmosphere').visible = false;
+            view.getLayerById('atmosphere').fog.enable = false;
+
+            setupLoadingScreen(viewerDiv, view);
+
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                var layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            // Define a VectorTilesSource to load Vector Tiles data from the geoportail
+            var mvtSource = new itowns.VectorTilesSource({
+                style: 'https://wxs.ign.fr/essentiels/static/vectorTiles/styles/PLAN.IGN/standard.json',
+                // We don't display mountains related data to ease visualisation
+                filter: (layer) => {
+                    const bati = (layer['source-layer'].includes('bati_surf') && layer.id.startsWith('bati surfacique'));
+                    return bati && layer.paint["fill-color"];
+                },
+            });
+
+            var mvtSourceColor = new itowns.VectorTilesSource({
+                style: 'https://wxs.ign.fr/essentiels/static/vectorTiles/styles/PLAN.IGN/standard.json',
+                // We don't display mountains related data to ease visualisation
+                filter: (layer) => {
+                    const f = layer['source-layer'].includes('bati_surf');
+                    return !f && !layer['source-layer'].includes('oro_')
+                     && !layer['source-layer'].includes('parcellaire')
+                }
+            });
+
+            var mvtLayer = new itowns.ColorLayer('MVT', {
+                source: mvtSourceColor,
+                addLabelLayer: true,
+            });
+
+            var mvtBuildingLayer = new itowns.FeatureGeometryLayer('WFS Building',{
```

Comment:
MVT Buildings instead of WFS buildings ?

Reply:
yes